### PR TITLE
feat: update container, cronjob and lifecycle clone command

### PR DESCRIPTION
**Pull request by pggb25**

### CODE REVIEW

**Review by erebe — APPROVED** [review]

File: cmd/container_clone.go [hunk 1]
```diff
@@ -26,148 +28,77 @@ var containerCloneCmd = &cobra.Command{
...
-			MaxRunningInstances: &container.MaxRunningInstances,
-			AutoPreview:         &container.AutoPreview,
-			Healthchecks:        container.Healthchecks,
+		req := qovery.CloneContainerRequest{
+			Name:          targetContainerName,
+			EnvironmentId: targetEnvironmentId,
 		}
 
-		createdService, res, err := client.ContainersApi.CreateContainer(context.Background(), targetEnvironment.Id).ContainerRequest(req).Execute()
+		clonedService, res, err := client.ContainersApi.CloneContainer(context.Background(), container.Id).CloneContainerRequest(req).Execute()
```
Comment (benjaminch) [inline cmd/container_clone.go:82]: _nitpick_: maybe we can replace service with container here:  `clonedService` => `clonedContainer`

File: cmd/container_clone.go [hunk 2]
```diff
@@ -26,148 +28,77 @@ var containerCloneCmd = &cobra.Command{
...
 
 		if err != nil {
-			utils.PrintlnError(err)
-
-			bodyBytes, err := io.ReadAll(res.Body)
-			if err != nil {
-				return
+			// print http body error message
+			if !strings.Contains(res.Status, "200") {
```
Comment (benjaminch) [inline cmd/container_clone.go:86]: _suggestion_: use dedicated `HTTP` field which is integer representing status code:
```
if res.StatusCode != 200 {}
```

File: cmd/cronjob_clone.go [hunk 3]
```diff
@@ -26,162 +28,77 @@ var cronjobCloneCmd = &cobra.Command{
...
+		clonedService, res, err := client.JobsApi.CloneJob(context.Background(), job.Id).CloneJobRequest(req).Execute()
 
 		if err != nil {
-			utils.PrintlnError(err)
-
-			bodyBytes, err := io.ReadAll(res.Body)
-			if err != nil {
-				return
+			// print http body error message
+			if !strings.Contains(res.Status, "200") {
```
Comment (benjaminch) [inline cmd/cronjob_clone.go:86]: _suggestion_: use `res.StatusCode` avoiding strings contain.

File: cmd/lifecycle_clone.go [hunk 4]
```diff
@@ -26,161 +28,77 @@ var lifecycleCloneCmd = &cobra.Command{
...
+		clonedService, res, err := client.JobsApi.CloneJob(context.Background(), job.Id).CloneJobRequest(req).Execute()
 
 		if err != nil {
-			utils.PrintlnError(err)
-
-			bodyBytes, err := io.ReadAll(res.Body)
-			if err != nil {
-				return
+			// print http body error message
+			if !strings.Contains(res.Status, "200") {
```
Comment (benjaminch) [inline cmd/lifecycle_clone.go:86]: _suggestion_: use res.StatusCode avoiding strings contain.

**Pull request merged**